### DISCUSSION TITLE
num_workers => max_workers

### DISCUSSION THREAD

**Discussion by SAgnius** [roadrunner-server/roadrunner#1272] [Q&A] [answered]
Dear colleagues,

do you plan to do something like dynamic max_workers instead static num_workers?

Regards and have a Great day

Agnius

**Comment by rustatian** [comment, marked as answer]
Hey @SAgnius 👋🏻 
Yep: https://github.com/roadrunner-server/roadrunner/issues/97

**Comment by attson** [comment]
Are there any concrete plans for this feature? 

I found out that this feature is set to low priority, Last active time is Feb 18, 2022.

When we use io operation, like with mysql or curl, if remote service cost a lot time. The worker is block until remote service return.

But the resources consumed by the machine are minimal. lower cpu, lower mem.

In fact, more workers can be served.

**Reply by rustatian** [reply]
We propose that everyone may attend (via chat on twitch) our RoadRunner planning sessions. (links in our Discord server) Unfortunately, we may miss some old tickets in favor of recently added ones. I'll put this ticket to the `2023.1` triage.
